org.virt_manager.virt-viewer.yml

```diff
@@ -0,0 +1,147 @@
+app-id: org.virt_manager.virt-viewer
+runtime: org.gnome.Platform
+runtime-version: "43"
+sdk: org.gnome.Sdk
+command: remote-viewer
+rename-icon: virt-viewer
```

Comment:
When i start the test build I get a warning:

```
$ flatpak run org.virt_manager.virt-viewer

(remote-viewer:2): Gtk-WARNING **: 09:52:54.270: Could not load a pixbuf from icon theme.
This may indicate that pixbuf loaders or the mime database could not be found.
```

I tried to strace it, and found that it was trying and failing to  open virt-viewer.png:
```
[pid 814477] openat(AT_FDCWD, "/app/share/icons/hicolor/256x256/apps/virt-viewer.png", O_RDONLY) = -1 ENOENT (No such file or directory)
```
 which makes me thing this `rename-icon` instruction might be the cause, most likely a result of the code calling ` gtk_window_set_default_icon_name("virt-viewer");` It might need a source change to avoid this.

Reply:
There's also `copy-icon` attribute that will copy the icon instead of renaming it.